Symfony/CS/Tests/Fixer/PSR2/LowercaseConstantsFixerTest.php

```diff
@@ -53,12 +44,6 @@ public function provideCases()
             array('<?php $x = "true story";'),
             array('<?php $x = "false";'),
             array('<?php $x = "that is null";'),
-            array('<?php $x = new True;'),
```

Comment:
Hmm, after more thought: The fixer should not change the code regardless of php version, so we should test it with php 7, too.